File: hw-triangle/hw-triangle.c
#include "stdio.h"

int main() {
    int a, b, c;
    scanf("%d%d%d", &a, &b, &c);

    (a + b > c && b + c > a && c + a > b)
        ? printf("the perimeter of the triangle is: %d\n", a + b + c)
        : printf("CANNOT make up a triangle\n");

    return 0;
}

File: hw-triangle/hw-triangle.py
# author = syqwq
# coding = utf-8

if __name__ == '__main__':
    a, b, c = (int(i) for i in input().split())

    if a + b > c and b + c > a and c + a > b:
        print("the perimeter of the triangle is: {}".format(a + b + c))
    else:
        print("CANNOT make up a triangle")
